src/commonControls/numberControl/NumberControlAPL.ts

```diff
@@ -175,3 +175,92 @@ export namespace NumberControlAPLPropsBuiltIns {
         };
     }
 }
+
+export type NumberControlAPLComponentStyle = 'touchForward';
```

Comment:
naming: lets go with 'textAndModalkeypad' or 'modalKeypad' or otherwise something that describes the actual rendering look and feel.  touchForward is an important concept but its not a rendering style per se.

Reply:
Updated to 'modalKeypad'